static/js/import_wizard2/utils/validation.ts

```diff
@@ -154,3 +154,12 @@ export function checkMappings(mappings: Mapping): Array<string> {
 
   return errors;
 }
+
+/**
+ * Returns whether a variable is valid.
+ * @param variable the variable to check
+ */
+export function isValidVariable(variable: string): boolean {
+  // checks that there are no spaces within the string.
+  return !/\s/.test(variable.trim());
```

Comment:
not sure if trim handles all the while spaces /t, /n etc

Reply:
looking at the documentation, looks like trim will remove all the whitespace from either end of the string
<img width="817" alt="Screenshot 2023-04-20 at 10 51 54 AM" src="https://user-images.githubusercontent.com/69875368/233447928-7ce6e76a-d4d9-45e1-984d-1f919aa10448.png">